### PR TITLE
FIX #2544: Target name gets highlighted

### LINKED ISSUE #2544: When scrolling to a target , The target name should get highlighted .
**Describe the bug**

In the sidebar when user clicks on Scrollable objects and button doesn't highlights their heading

**To Reproduce**

Steps to reproduce the behaviour:

1. View any event
2. Click on *Scrollable object like Tickets or Getting Here* .
3. They dont get highlighted and info is highlighted rather.

**Expected behaviour**

They should be highlighted as 'info'
**Screenshot**

image

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Chrome
 - Version: 73.0.3

**Additional context**
I am on it

### PR DESCRIPTION
**Pull request by geetchoudhary**
Fixes #2544 

#### Short description of what this resolves:
In the sidebar when the user clicks on scrollable links the correct `{{scroll-to}}` gets highlighted.

#### Video:
mp4gifgif

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by geetchoudhary** [comment]
@kushthedude , PTAL. Thanks :smile:

File: app/components/public/side-menu.js [hunk 1]
```diff
@@ -3,12 +3,18 @@ import { computed } from '@ember/object';
 import moment from 'moment';
 
 export default Component.extend({
+  selectedIndex: undefined,
```
Comment (kushthedude) [inline app/components/public/side-menu.js]: Can you make it `index`?

Comment (geetchoudhary) [reply app/components/public/side-menu.js]: We can do this but having  
```
actions: {
    changeIndex(index) {
      this.set('Index', index);
    }
  }
```
will be very confusing for future issues, if any.

**Comment by CosmicCoder96** [comment]
Stale, also we are approaching it the wrong way, maybe scroll to should be replaced with something that works, no point of using a library, and then manually writing an action, which the library was supposed to take itself.

**Pull request closed without merging**